bundles/io/org.eclipse.smarthome.io.console/src/main/java/org/eclipse/smarthome/io/console/internal/extension/ItemConsoleCommandExtension.java

```diff
@@ -31,6 +32,8 @@
     private static final String SUBCMD_LIST = "list";
     private static final String SUBCMD_CLEAR = "clear";
     private static final String SUBCMD_REMOVE = "remove";
+    private static final String SUBCMD_ADDTAG = "addTag";
+    private static final String SUBCMD_REMTAG = "remTag";
```

Comment:
Hm, what do you think about "rmTag"? It is more like the Unix command "rm" to "remove" something.